test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
To be clear the breaking change here is this is __no longer true__

> when an app/module has been started, and registering another initializer it will __automatically be invoked__

This raises the question of what to do if you want this behavior.

Well I would say since you have to have had a reference to your module/app (before this breaking change) you can simply invoke the function with the app/module as the callee of the method with the options you would like via call or apply.

How does this sound to you @derickbailey